File: src/ultra_light/echo.cpp
/* Light builtins's (Ultra Light)
 * echo - display a line of text
 * A lightweight repository of useful shell commands
 * GitHub: https://www.github.com/awesomelewis2007/light_builtins
 * License: GNU General Public License v3.0
*/

#include <iostream>
#include <string>

int main (int argc, char *argv[]) {
    std::string str = argc > 1 ? argv[1] : "";
    std::cout << str << std::endl;
    return 0;
}

File: src/python/echo.py
#!/usr/bin/env python3
# Light builtins's (Python version)
# echo - print arguments to the standard output
# A lightweight repository of useful shell commands
# GitHub: https://www.github.com/0x4248/light_builtins
# Licence: GNU General Public License v3.0
# By: 0x4248

import os
import sys


def main(args):
    print(" ".join(args))
    return 0


if __name__ == "__main__":
    sys.exit(main(sys.argv[1:]))
